### PR TITLE
Add GitHub Actions bot workflow to fix code linting from a PR comment

### PR DESCRIPTION
**Pull request by vladsavelyev**
Fixes https://github.com/ewels/MultiQC/issues/1988

A bot command triggered on a comment "@multiqc-bot fix linting". Inspired by [nf-core/tools](https://github.com/nf-core/tools/blob/master/.github/workflows/fix-linting.yml), though installs and runs pre-commit hooks on all files instead of running separate actions, so we control all linting through the single file `.pre-commit-config.yaml`.

I also updated the linters' versions.

### CODE REVIEW

**Comment by vladsavelyev** [comment]
@multiqc-bot fix linting

File: CSP.txt [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
 script-src 'self'
 
+    # 1.17
+    'sha256-krKzkLmKjisEgw0YGKglUFqLmEh6sK08Qw6xPmmo/10=' # ////////////////////////////////////////////////// Base JS for MultiQC Reports//
+    'sha256-8k3VwupzY7v2YyNWjgl7LjJRpDHztn+213IxxMtlhGU=' # ////////////////////////////////////////////////// HighCharts Plotting Code/////
+    'sha256-F/Ns5XO/YJO5xM4MiEoOb0Oo02hP7aTlfRywlFde9G4=' # ////////////////////////////////////////////////// MultiQC Report Toolbox Code//
+    'sha256-lD87d8oI1kNPJlsmI02I4WwpZwxfpFZnvq5MdsR3yaE=' # // Return JS code required for plotting a single sample// RSeQC plot. Attempt to
+    'sha256-Nbg17DNbGUiybFNgr6wbIU1w5j833X0XIK/iycwTXDQ=' # // Javascript for the FastQC MultiQC Mod///////////////// Per Base Sequence Cont
+    'sha256-Y19Ab/rDOuOgOuoftOFGSCo6C6kLjM/LSa2eQgrOCUM=' # // Javascript for the FastQC MultiQC Mod///////////////// Per Base Sequence Cont
+
```
Comment (ewels) [inline CSP.txt:13]: Not sure if this is meant to be here?

**Review by ewels — APPROVED** [review]
Great! 👏🏻

File: CSP.txt (hunk 1 above)
Comment (ewels) [reply CSP.txt:13]: ah right, it's because Prettier messed with the files. Gotcha 👍🏻

**Comment by ewels** [comment]
@multiqc-bot changelog

File: .github/workflows/fix-linting.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+      - name: Install pre-commit
+        run: |
+          python -m pip install --upgrade pip
+          pip install pre-commit
+          pre-commit install
+
+      - name: Run pre-commit
+        run: |
+          pre-commit run --all-files
```
Comment (ewels) [inline .github/workflows/fix-linting.yml:37]: I think this needs an `|| true` as if it does anything it raises an exit code 1 and the rest of the workflow is cancelled 😅

**Pull request merged**